sdk/core/azure-core/src/http/curl/curl_connection_private.hpp

```diff
@@ -26,13 +25,15 @@
 #pragma warning(pop)
 #endif
 
+typedef struct x509_store_ctx_st X509_STORE_CTX;
```

Comment:
This doesn't follow our naming guidelines.

Reply:
It's not our type - this is a forward declaration of an OpenSSL type and it cannot follow our naming guidelines.